### PR TITLE
Add nirum syntax highlighting to docs target

### PR DESCRIPTION
**Pull request by qria**
image

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ To be released.
     [[#297], [#304] by Jeong Ukjae]
  -  Added `opengraphs` option for [OpenGraph] objects on docs.
     [[#283], [#305] by GyuYong Jung]
- -  Added syntax highlighting.
-    [[#310], [#311] by MinJune Kim]
+ -  Added syntax highlighting including nirum.
```
Comment (dahlia) [inline CHANGES.md]: nirum → Nirum

File: src/Nirum/Targets/Docs.hs [hunk 2]
```diff
@@ -635,11 +636,81 @@ window.addEventListener('load', function () {
...
+    var NIRUM_IDENTIFIER_MODE = {
+        [className]: 'title',
+        [begin]: NIRUM_IDENTIFIER_RE,
+        [relevance]: 0,
+    };
+
+    return {
+        [keywords]: {
+            keyword: 'record enum unboxed type union service' +
+                    ' import throws as default',
```
Comment (dahlia) [inline src/Nirum/Targets/Docs.hs]: Vertical aligment should be adjusted.

File: src/Nirum/Targets/Docs.hs [hunk 3]
```diff
@@ -635,11 +636,81 @@ window.addEventListener('load', function () {
...
+            built_in: 'bigint decimal int32 int64 float32 float64' +
+                    ' text binary datetime date bool uuid uri',
```
Comment (dahlia) [inline src/Nirum/Targets/Docs.hs]: Verical aligment should be adjusted.

File: src/Nirum/Targets/Docs.hs [hunk 4]
```diff
@@ -635,11 +636,81 @@ window.addEventListener('load', function () {
...
+hljs.initHighlightingOnLoad();
+|] undefined :: Javascript)
+
 compilePackage' :: Package Docs -> Map FilePath (Either Error BS.ByteString)
 compilePackage' pkg = unions
     [ fromList
         [ ("style.css", Right $ encodeUtf8 css)
         , ("nirum.js", Right $ encodeUtf8 javascript)
+        , ("nirumHighlight.js"
+          , Right $ encodeUtf8 $ TL.toStrict nirumHighlightJavascript)
```
Comment (dahlia) [inline src/Nirum/Targets/Docs.hs]: The closing parenthesis should be aligned to the opening parenthesis and commas.

File: src/Nirum/Targets/Docs.hs [hunk 5]
```diff
@@ -635,11 +636,81 @@ window.addEventListener('load', function () {
...
+});
+hljs.initHighlightingOnLoad();
+|] undefined :: Javascript)
+
 compilePackage' :: Package Docs -> Map FilePath (Either Error BS.ByteString)
 compilePackage' pkg = unions
     [ fromList
         [ ("style.css", Right $ encodeUtf8 css)
         , ("nirum.js", Right $ encodeUtf8 javascript)
+        , ("nirumHighlight.js"
```
Comment (dahlia) [inline src/Nirum/Targets/Docs.hs]: If a pair of parentheses contain multiple lines a whitespace should be placed right after the opening parenthesis.

**Comment by qria** [comment]
@dahlia fixed as requested

File: src/Nirum/Targets/Docs.hs [hunk 6]
```diff
@@ -635,11 +636,82 @@ window.addEventListener('load', function () {
...
+        [relevance]: 0,
+    };
+
+    return {
+        [keywords]: {
+            keyword: 'record enum unboxed type union service' +
+                      ' import throws as default',
+            built_in: 'bigint decimal int32 int64 float32 float64' +
+                      ' text binary datetime date bool uuid uri',
+            literal: 'true false',
```
Comment (admire93) [inline src/Nirum/Targets/Docs.hs]: I think there are no literal called `true` or `false` in Nirum IDL

Comment (qria) [reply src/Nirum/Targets/Docs.hs]: it is gone now.

**Pull request merged**